### ISSUE TITLE
Legacy APNS push notifications

### ISSUE THREAD

**Issue by sglessard** [mac-cain13/notificato#67]
Hello,

Does notificato have a chance to migrate to the Apple replacement HTTP/2 API?
https://developer.apple.com/news/?id=uzyxiriy

Thanks
All the best

**Comment by pedro380085** [comment]
Yes it stopped working for me too. At least archive this library since it no longer works.
